apps/frontend/src/pages/projects/libs/helpers/get-relative-date-label.helper.tsx

```diff
@@ -0,0 +1,38 @@
+import { getDifferenceInDays } from "~/libs/helpers/helpers.js";
+
+import { ColorClass, ColorThreshold } from "../enums/enums.js";
+
+type RelativeDateLabel = {
+	colorClass: string;
+	label: string;
+} | null;
+
+const getRelativeDateLabel = (
```

Comment:
We need to use date fns for calculating relative date. This is the example from the main page:

```
formatDistance(subDays(new Date(), 3), new Date(), { addSuffix: true })
//=> "3 days ago"
```

Also we need to calculate date and color separately.